### DISCUSSION TITLE
use-permission in manifest.xml 관련하여

### DISCUSSION THREAD

**Discussion by cotingmaster** [velopert/dealing-with-react-native#1] [Q&A] [answered]
안녕하세요, 정말 너무 좋은 책으로 찾아와주셔서 감사합니다.

요즘 혹시 react-native-image-picker 에서는 manifest.xml 파일에 use permissions을 적어주지 않아도 되는 것인지 확인차 질문드립니다.
책 내용에서는 p467 상단에 있는 내용이고,  
https://github.com/react-native-image-picker/react-native-image-picker
여기 링크에서 버전 4.3.0부터 manifest에 아무것도 쓰지 말라고 하는 것 같아서요.

제가 정말 궁금한 내용은 이것이 아니라 다음에 올릴 질문인데요, 혹시 제가 이것을 안해서 이따가 쓸 오류가 나타나는 건지 확인해보려고 이 질문을 드립니다. 메인 질문을 바로 다시 작성하도록 할게요. 감사합니다.

**Comment by velopert** [comment, marked as answer]
과거엔 필요했었지만 이젠 필요없는것이 맞습니다. 
책에 나와있는 것 처럼 바로 저장 / SD 카드 읽기를 할 때만 필요한데,
책의 실습에는 필요없는 내용이니 헷갈리는걸 방지하기 위해 지우도록 하겠습니다.
